README.md

```diff
@@ -1,8 +1,8 @@
-# Colcon cc-build
+# ROS2 Cross Compile
 
-Colcon plugin for cross-compilation
+A package for cross compiling ROS2 packages.
 
-## Install prerequisites
+## Prerequisites
```

Comment:
I can't comment on lines 30-31, but I think they should be moved out of this section to the source installation section.

They cannot be executed unless the repo is cloned first.

Reply:
which lines?